chia/wallet/puzzles/clawback/drivers.py

```diff
@@ -121,17 +122,15 @@ def create_merkle_solution(
 
 
 def match_clawback_puzzle(
```

Comment:
I would expect some callers of this function to not have full mypy checking. Is that a concern of yours?
I haven't looked into it deeply, but `wallet_tools` and `did_wallet` looks like risks

Reply:
This is part of the type checking effort. If you have specific cases we need to check, please let us know.